### PR TITLE
added spectral domain functions in spectral.py, attempt #2

### PR DESCRIPTION
**Pull request by rdgao**
fixed a failing test in spectral.psd: spectrogram was not handling a None parameter

### CODE REVIEW

**Comment by TomDonoghue** [comment]
PEP8 OR GO HOME!

File: neurodsp/tests/test_spectral.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
+"""
+test_timefreq.py
+Test functions in the time-frequency analysis module
+"""
+
+import pytest
```
Comment (TomDonoghue) [inline neurodsp/tests/test_spectral.py]: You don't need to import pytest for a test file.

Comment (srcole) [reply neurodsp/tests/test_spectral.py]: True. This should be changed for all test files.

File: neurodsp/tests/test_spectral.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
+"""
+test_timefreq.py
```
Comment (srcole) [inline neurodsp/tests/test_spectral.py]: update the header for your test file

**Pull request merged**